cmd/kuberhealthy/kuberhealthyCheck.go

```diff
@@ -45,4 +47,8 @@ type KuberhealthyCheck interface {
 	// When the check completes and returns, we assume it is done shutting
 	// down.
 	Shutdown() error
+	// GenOwnerRef can generate the OwnerReference for objects(pod/cr)
```

Comment:
This interface was from Kuberhealthy v1.  In v1 we had lots of check structs that would be thrown in from different packages.  In v2, we only have one!  That one is the external checker, which then runs all other checks.

I think it's okay to add this to the interface for now, but I wanted to document the intent to remove this interface.